### PR TITLE
card integration

### PR DESCRIPTION
**Pull request by Scottjrw**
integrated card for customers with backend (params mmp)

### CODE REVIEW

File: app/account_info/accountInfo.js [hunk 1]
```diff
@@ -17,30 +17,43 @@ angular.module('myApp.accountInfo', [
...
+          // handle error here
+          console.log(err);
+        });
+
       });
     }
     $scope.maskCard = function(number) {
       // 1111 2222 3333 4444 to 1111-xxxx-xxxx-4444
-      return `${number.substring(0,4)}-XXXX-XXXX-${number.substring(11,15)}`;
+      return `${number.substring(0,4)}-XXXX-XXXX-${number.substring(12,16)}`;
```
Comment (suiyoubi) [inline app/account_info/accountInfo.js]: are you sure..?

Comment (Scottjrw) [reply app/account_info/accountInfo.js]: very sure

Comment (chenyirex) [reply app/account_info/accountInfo.js]: damn we have the XXXX, really good thinking here

File: app/account_info/accountInfo.js [hunk 2]
```diff
@@ -17,30 +17,43 @@ angular.module('myApp.accountInfo', [
...
     const getReviewsUrl = `${$rootScope.url}/reviews/username/${username}`;
+    const createCardUrl = $rootScope.url + "/cards/create";
+    const userCardUrl = `${url}/cards/${userType}/${username}`;
     $scope.addressUrl = `${url}/addresses`;
     $scope.address = {street:"", city:"", province:"", postal_code:"", country:""};
 
     $scope.cards = [
-      {card_number: '1234123412341234', card_holder_name:'haus', expire_date: '2008/01/01'},
-      {card_number: '4444333322221111', card_holder_name:'rex', expire_date: '2088/01/01'},
+      // {card_number: '1234123412341234', card_holder_name:'haus', expire_date: '2008/01/01'},
```
Comment (chenyirex) [inline app/account_info/accountInfo.js]: is there a reason that we would still need these hard-coded cards?

Comment (Scottjrw) [reply app/account_info/accountInfo.js]: removed

**Pull request merged**